controllers/ingressUpdater.go

```diff
@@ -210,10 +236,21 @@ func (i *ingressUpdater) upsertDoguIsStartingIngressObject(ctx context.Context,
 
 func (i *ingressUpdater) upsertDoguIngressObject(ctx context.Context, cesService CesService, service *corev1.Service) error {
 	log.FromContext(ctx).Info(fmt.Sprintf("dogu is ready -> update ces service ingress object for service [%s]", service.GetName()))
+	serviceRewrite, err := cesService.generateRewriteConfig()
+	if err != nil {
+		return err
+	}
+
+	// This should overwrite the `Accept-Encoding: "gzip"` header that browsers send.
+	// Gzipping by dogus is a problem because it prevents the warp menu from being injected.
+	encodingOverwrite := "proxy_set_header Accept-Encoding \"identity\";"
+
+	configurationSnippet := fmt.Sprintf("%s", encodingOverwrite)
```

Comment:
🚫 **[checkstyle]** <gosimple> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>S1025: the argument is already a string, there's no need to use fmt.Sprintf